src/components/shared/Layout.js

```diff
@@ -192,18 +192,19 @@ export default class Layout extends React.Component {
   }
 
   render() {
+    const { children, location } = this.props;
     return (
       <>
         <SiteMetadata />
         <UserContext.Provider value={this.state.user}>
           <StoreContext.Provider value={this.state.store}>
             <Header />
-            {!this.state.user.profile.name && <CTA />}
-            <Main>{this.props.children}</Main>
+            {!user.profile.name && <CTA />}
```

Comment:
Oops, we missed one:

```suggestion
            {!this.state.user.profile.name && <CTA />}
```

Reply:
Fixed.